### PR TITLE
[nnc] tensorexpr for quantized/aten::upsample_nearest2d

### PR DESCRIPTION
**Pull request by IvanKobzarev**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #70193
* __->__ #70189
* #70188

Differential Revision: [D33238092](https://our.internmc.facebook.com/intern/diff/D33238092)

### CODE REVIEW

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33238092).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33238092).

File: torch/csrc/jit/tensorexpr/operators/quantization.cpp [hunk 1]
```diff
@@ -721,6 +721,51 @@ Tensor computeUpsampleNearest2d(
...
+  };
+  auto e = body_func(VarVectorToVarHandleVector(args));
+  BufPtr buf = alloc<Buf>(
+      "quantize_upsample_nearest2d",
+      ExprHandleVectorToExprVector(outputShape),
+      Dtype(*outputType),
+      nullptr,
+      c10::nullopt,
+      A.node()->qscale(),
+      A.node()->qzero());
```
Comment (ZolotukhinM) [inline torch/csrc/jit/tensorexpr/operators/quantization.cpp:760]: Similarly to the previous PR, this code would probably be slightly more elegant if we used `Compute`.

**Review by ZolotukhinM — APPROVED** [review]
👍

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33238092).

**Pull request closed without merging**